isobus/src/isobus_virtual_terminal_server.cpp

```diff
@@ -1168,18 +1168,29 @@ namespace isobus
 											}
 											break;
 
+											case isobus::VirtualTerminalObjectType::PictureGraphic:
+											{
+												targetObject->set_width(newWidth);
+											}
+											break;
+
 											case VirtualTerminalObjectType::Animation:
 											case VirtualTerminalObjectType::Button:
 											case VirtualTerminalObjectType::Container:
+											case VirtualTerminalObjectType::InputBoolean:
 											case VirtualTerminalObjectType::InputList:
+											case VirtualTerminalObjectType::InputString:
+											case VirtualTerminalObjectType::InputNumber:
 											case VirtualTerminalObjectType::OutputArchedBarGraph:
 											case VirtualTerminalObjectType::OutputEllipse:
 											case VirtualTerminalObjectType::OutputLine:
+											case VirtualTerminalObjectType::OutputLinearBarGraph:
 											case VirtualTerminalObjectType::OutputList:
 											case VirtualTerminalObjectType::OutputNumber:
 											case VirtualTerminalObjectType::OutputPolygon:
 											case VirtualTerminalObjectType::OutputRectangle:
 											case VirtualTerminalObjectType::OutputString:
+											case VirtualTerminalObjectType::ScaledGraphic:
```

Comment:
Scaled graphic doesn't list change size command as an allowed command in ISO 11783-6, and isn't an output field object, so IDK if that one applies... same thing with picture graphic, if I am reading it correctly (which, maybe I'm not) that one should only allow "change attribute" and "get attribute"?